man/systemd.network.xml

```diff
@@ -2317,6 +2317,16 @@
             the value of a received bit by majority rule. When unset, the kernel's default will be used.</para>
           </listitem>
         </varlistentry>
+        <varlistentry>
+          <term><varname>ListenOnly=</varname></term>
+          <listitem>
+            <para>Takes a boolean. When <literal>yes</literal>, listen-only mode is enabled. When the
+            interface is in listen-only mode, the interface does not transmit CAN frames or not send
```

Comment:
" or not send" needs rewording

Reply:
How about "the interface neither transmit CAN frames or not send ACK bit."?

Reply:
"nor send ACK bit" sounds better to me...

Reply:
OK. Will update. Thanks.